CHANGELOG.md

```diff
@@ -1,5 +1,7 @@
 ## 1.0.1
```

Comment:
How about 1.1 version? I think it is already something more than small bugfix.

Reply:
fully agree